### PR TITLE
added directories for homepage

### PR DESCRIPTION
**Pull request by pivilartisant**
We create directories in order to render pages. 
This is for development purposes and will be modified in the future to have unique routes

### CODE REVIEW

**Review by Thykof — APPROVED** [review]

File: web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function SendCoins() {
+  return <h1>Send coins </h1>;
+}
```
Comment (maxime-carabina) [inline web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]:
```suggestion
function SendCoins() {
  return <h1>Send coins </h1>;
}

export default SendCoins;
```

Comment (pivilartisant) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: is it best practice ?

Comment (maxime-carabina) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: Yes! For pages is better to have this format. But if it's a component, we have `import { MyComponent } from "......";`.

Comment (maxime-carabina) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: What did you think @mazmassa ?

File: web-frontend/src/main.tsx [hunk 2]
```diff
@@ -37,6 +38,7 @@ const router = createBrowserRouter(
       <Route path="account-create-step-one" element={<StepOne />} />
       <Route path="account-create-step-two" element={<StepTwo />} />
       <Route path="account-create-step-three" element={<StepThree />} />
+      <Route path="homepage-transfer-send" element={<SendCoins />} />
```
Comment (mazmassa) [inline web-frontend/src/main.tsx:41]: The url is: homepage-transfer-send and we are rendering `<SendCoins>`. Please, let's keep a standard for that as also mentioned here: https://discord.com/channels/@me/1100786679486369844/1113023630364901406

File: web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx (hunk 1 above)
Comment (mazmassa) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: Let me just understand first why are we exporting one <h1> code!? 🤔

Comment (pivilartisant) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: it's just a placeholder :)

Comment (mazmassa) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: What you mean is that will become a more complex component or a page!?

File: web-frontend/src/main.tsx (hunk 2 above)
Comment (mazmassa) [reply web-frontend/src/main.tsx:41]: Fist, why homepage!? Then, if you want to keep your standard it should be `send-coins`. Finally, it's a huge name that does not match with the component name you are rendering.

Change that for: /send-coins and render the `<SendCoins>` component. So, your URL will match with the component you are rendering.

/apple -> </Apple>
/number -> </Number>
/something -> </Something>

and not 

/apple -> </Tomato>

File: web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx (hunk 1 above)
Comment (pivilartisant) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: This will be a component !

Comment (mazmassa) [reply web-frontend/src/pages/Home/TransferCoins/SendCoins/SendCoins.tsx:3]: Ok! Got it!

File: web-frontend/src/main.tsx (hunk 2 above)
Comment (pivilartisant) [reply web-frontend/src/main.tsx:41]: understood ! thank you for the clarifications

File: web-frontend/src/main.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ import StepTwo from './pages/CreateAccount/StepTwo.tsx';
 import StepOne from './pages/CreateAccount/StepOne.tsx';
 import StepThree from './pages/CreateAccount/StepThree.tsx';
 import mockServer from './mirage/server.ts';
+import { SendCoins } from './pages/Home/TransferCoins/SendCoins/SendCoins.tsx';
```
[the remark this answers is not in the document]
Comment (pivilartisant) [reply web-frontend/src/main.tsx]: changed !

File: web-frontend/src/main.tsx (hunk 2 above)
Comment (pivilartisant) [reply web-frontend/src/main.tsx]: changed !

**Review by Thykof — APPROVED** [review]

File: web-frontend/src/main.tsx [hunk 4]
```diff
@@ -37,6 +38,7 @@ const router = createBrowserRouter(
       <Route path="account-create-step-one" element={<StepOne />} />
       <Route path="account-create-step-two" element={<StepTwo />} />
       <Route path="account-create-step-three" element={<StepThree />} />
+      <Route path="send-coins" element={<SendCoins />} />
```
Comment (mazmassa) [inline web-frontend/src/main.tsx:41]: 👏 beautiful!!!

**Pull request merged**